lib/prepareReturnValue.mjs

```diff
@@ -35,11 +35,11 @@ export default function prepareReturnValue(stylelintResults, maxWarnings, format
 		// TODO: Deprecated. Remove in the next major version.
 		get output() {
 			if (!this._outputWarned) {
```

Comment:
suggestion
minor

Could you change the signature of `prepareReturnValue` to be
```
 * @param {StylelintResult[]} stylelintResults
 * @param {Formatter} formatter
 * @param {object} opts
 * @param {string} opts.cwd
 * @param {maxWarnings} opts.maxWarnings
 * @param {boolean} opts.quietDeprecationWarnings
```
so that we can add `&& !opts.quietDeprecationWarnings` to the condition?

Reply:
Your suggestion makes sense, but it's out of the scope of this PR, which is just a refactoring.

I want to work on that after merging this PR.

Reply:
Just opened follow-up PR #7837